easybuild/easyblocks/b/boost.py

```diff
@@ -89,6 +89,8 @@ def patch_step(self):
     def configure_step(self):
         """Configure Boost build using custom tools"""
 
+        configopts = "--prefix=%s" % self.objdir
```

Comment:
oh, and do this *after* `self.objdir` is defined below

so, this wasn't tested? :)